README.md

```diff
@@ -18,7 +18,7 @@ Add this to your `package.json`:
 ```json
 {
   "scripts": {
-    "start": "auto-reveal",
+    "start": "auto-reveal start",
```

Comment:
Is this not a breaking change? we should be able to make this not breaking by making start the "default" command

Reply:
This _is_ indeed a breaking change.